src/Twilio.TaskRouter/Model/Filter.cs

```diff
@@ -17,6 +17,12 @@ public Filter()
         [DataMember(EmitDefaultValue = false, Name="friendly_name")]
         public string FriendlyName { get; set; }
 
+        [DataMember(EmitDefaultValue = false, Name = "filter_friendly_name")]
+        public string FilterFriendlyName {
+            get { return null; }

```

Comment:
I did this because we only want to marshal friendly_name and by returning this.FriendlyName, it ends up marshalling both filter_friendly_name and friendly_name in the workflow configuration.